packages/server/prisma/schema.prisma

```diff
@@ -65,6 +67,90 @@ model Team {
   @@unique([gameId, name], map: "Team_gameId_name_unique_constraint")
 }
 
+model Profile {
+  id                Int             @id @default(autoincrement())
+  player            Players[]
```

Comment:
@remiriv si on veut avoir un seul player par profile il faut utiliser un `@unique`. https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations#overview

```
profileId   Int?  @unique
```

Sinon renommons ce champs en `players`.

Reply:
@remiriv il faut aussi changer `player` de Profile en:

`player            Players?`